### DISCUSSION TITLE
后台挂不住

### DISCUSSION THREAD

**Discussion by DLGum** [2dust/v2rayNG#3533] [General]
在提出问题前请先自行排除服务器端问题，同时也请通过搜索确认是否有人提出过相同问题。

### 预期行为
解除后台限制，解除电池优化，锁后台卡片后，可以稳定挂住。

### 实际行为
挂不住。灭屏一会就得重开线路。
手机华为鸿蒙4。

**Comment by wukan1986** [comment]
Android官方常驻后台的方法是“常驻通知”。

所以可以试试开启相应权限。

**Comment by 2dust** [comment]
后台每个手机都不同，你去网上搜索下。
如果还是不行，就换一个app试一试。
btw，或者换一个手机
